conf.py

```diff
@@ -229,3 +229,11 @@
     ('index', 'pythondevelopersguide', u"Python Developer's Guide Documentation",
      [u'Brett Cannon'], 1)
 ]
+
+# ignore linkcheck anchors for /#/$ANCHOR since it is used for
+# dynamic pages such as http://buildbot.python.org/all/#/console
+# http://www.sphinx-doc.org/en/stable/config.html?highlight=linkcheck#confval-linkcheck_anchors_ignore
+linkcheck_anchors_ignore = [
+    # match any anchor that starts with a '/' since this is an invalid HTML anchor
+    '\/.*',
+]
```

Comment:
Missing newline at the end of the file.

Reply:
Ugh, even my linter yelled at me and I ignored it. Thank you :)